### PR TITLE
Don't redirect user to login page if logged in

### PR DESCRIPTION
**Pull request by xaoseric**
If a user is logged in, this prevents the login method from being called again and simply redirects the user.

### CODE REVIEW

File: 01-Login/app/Http/Controllers/Auth/Auth0IndexController.php [hunk 1]
```diff
@@ -13,7 +13,11 @@ class Auth0IndexController extends Controller
      */
     public function login()
     {
-        return \App::make('auth0')->login(null, null, ['scope' => 'openid name email email_verified'], 'code');
+        if (!auth()->check()) {
```
Comment (joshcanhelp) [inline 01-Login/app/Http/Controllers/Auth/Auth0IndexController.php]: Let's switch this to `auth()->check()` and redirect if that's true.

File: 01-Login/app/Http/Controllers/Auth/Auth0IndexController.php [hunk 2]
```diff
@@ -13,7 +13,11 @@ class Auth0IndexController extends Controller
...
+            return \App::make('auth0')->login(null, null, ['scope' => 'openid name email email_verified'], 'code');
+        } else {
```
Comment (joshcanhelp) [inline 01-Login/app/Http/Controllers/Auth/Auth0IndexController.php]: No need for an else

**Review by joshcanhelp — CHANGES_REQUESTED** [review]
@xaoseric - Thank you once again! Minor change in the flow here and I'll get this merged 👍

**Comment by xaoseric** [comment]
The changes requested have been made, I also noticed that there was an else in the profile method, fixed that up as well

**Review by joshcanhelp — APPROVED** [review]
@xaoseric Thank you!

Is there a reason why `auth()` is used in one place and `\Auth` in another?

**Comment by evansims** [comment]
Suggested changes have been rolled into #48, thank you

**Pull request closed without merging**